### PR TITLE
Fs 136

### PR DESCRIPTION
**Pull request by ljgreenhill**
Merge Adler's command changes. Removed FaultCheckCommand.

### CODE REVIEW

File: src/Commands/RockblockCommand.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    return (this->arg_2[3] << 24) | (this->arg_2[2] << 16) | (this->arg_2[1] << 8) | (this->arg_2[0]);
+}
+
+RockblockCommand RockblockCommand::commandFactory(RawRockblockCommand raw)
+{
+    // Create Specific Child Class of Rockblock command depending on the OP Code
+    uint16_t op_code = raw.get_f_opcode();
+    if (op_code == opcodes::mission_mode) {
+        return MissionModeCommand(raw);
+    } else if (false) { // TODO After SFRField Merge
```
Comment (Duncan-McD) [inline src/Commands/RockblockCommand.cpp:29]: SFROverrideCommand won't run

File: src/constants.hpp [hunk 2]
```diff
@@ -61,298 +61,84 @@ namespace constants {
...
+            constexpr uint16_t fault_check_mag_x = 65522;
+            constexpr uint16_t fault_check_mag_y = 65523;
+            constexpr uint16_t fault_check_mag_z = 65524;
+            constexpr uint16_t fault_check_gyro_x = 65525;
+            constexpr uint16_t fault_check_gyro_y = 65526;
+            constexpr uint16_t fault_check_gyro_z = 65527;
+            constexpr uint16_t fault_check_temp_c = 65528;
+            constexpr uint16_t fault_check_solar_current = 65529;
+            constexpr uint16_t fault_check_voltage = 65530;
+        } // namespace opcodes
```
Comment (Duncan-McD) [inline src/constants.hpp:100]: Do we still want these fault commands

**Review by Duncan-McD — CHANGES_REQUESTED** [review]

File: src/Control Tasks/RockblockControlTask.cpp [hunk 3]
```diff
@@ -394,44 +393,41 @@ void RockblockControlTask::dispatch_process_command()
...
+                // Comand is Valid - Will be added to list to be Executed During CommandMonitor Execute
+                sfr::rockblock::processed_commands.push_back(processed);
                 sfr::rockblock::waiting_command = true;
-            } else if (sfr::rockblock::raw_commands.back().opcode[0] == 'F' && sfr::rockblock::raw_commands.back().opcode[1] == 'L') {
+            } else if (new_raw_command.opcode[0] == 'F' && new_raw_command.opcode[1] == 'L') {
                 Serial.println("SAT INFO: flush confirmed");
                 sfr::rockblock::flush_status = false;
+            } else {
+                // Invalid Command Recieved
+                // TODO: What Goes Here @Lauren
```
[the remark this answers is not in the document]
Comment (ljgreenhill) [reply src/Control Tasks/RockblockControlTask.cpp:427]: For an invalid command, we do nothing. I will just add a print for debugging purposes.

**Review by Duncan-McD — APPROVED** [review]
Post changes - This looks good to me

**Pull request merged**